docs/source/install.rst

```diff
@@ -0,0 +1,570 @@
+Install LLM Operator
+====================
+
+We provide a Helm chart for installing LLM Operator. You can obtain the Helm chart from our repository and install.
+
+.. code-block:: console
+
+   # Logout of helm registry to perform an unauthenticated pull against the public ECR
+   helm registry logout public.ecr.aws
+
+   helm upgrade --install \
+     --namespace <namespace> \
+     --create-namespace \
+     llm-operator oci://public.ecr.aws/cloudnatix/llm-operator-charts/llm-operator \
```

Comment:
should it be `oci://public.ecr.aws/cloudnatix/llmariner-charts/llm-operator`

Reply:
Yeah I'll do all the renaining later.